### ISSUE TITLE
Scalability with limited amount of UDP port for kraken

### ISSUE THREAD

**Issue by sooxt98** [MixinNetwork/kraken#23]
I saw your blog post, it stated each participants will consume 1 udp port on kraken, which means i could only have maximum of 48127  participants online simultaneously (common available udp port range 1024 to 49151), How to scale it up to allow more than that amount of users? 

Here's the idea i got, make a daemon that actively monitor the usage of the kraken UDP port availability, and magically use a Load Balancing to distribute the traffic to others instance

**Comment by arabtion** [comment]
hey bro can u help me install kraken on vps nginx

**Comment by cedricfung** [comment]
Yes, we use many instances to balance the traffic. That we have a monitor API to monitor all the kraken instances server load, and direct participants in the same room to the same instance.

**Comment by sooxt98** [comment]
@cedricfung Thanks for the quick reply, is there any guide in using the monitor feature of kraken?

**Comment by cedricfung** [comment]
The monitor is another piece of code, that monitors the instance resource usage using Google Compute Engine API. We didn't open source those code yet, because they are coupled with our main repo. And we have the plan to make the monitor open source, but busy at other stuffs.

**Comment by arabtion** [comment]
@sooxt98 Why do I not see anyone? I checked in twice to make sure, but there is something wrong
look screen
https://prnt.sc/1281pou please help me  all ppl not know this problem
